docs/source/releasenotes/3.2.rst

```diff
@@ -0,0 +1,16 @@
+.. _Version32ReleaseNotes:
+
+3.2 Release Notes
+==================
+
+Koverse version 3.2 introduces the ability to apply security label parsers to transform outputs.
```

Comment:
I think we should specify JVM based transforms. Since Python is a separate ticket.

Reply:
changed